wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -930,17 +930,18 @@ body.page-slug-whats-on h1.entry-title {
 #wp20-events-filter {
 	position: relative;
 	display: flex;
-	flex-wrap: nowrap;
+	justify-content: flex-end;
+	flex-wrap: wrap;
+	flex-basis: 45%;
 	box-sizing: border-box;
-	min-width: 325px;
 	margin-bottom: 0;
 	padding-right: 0;
 }
 
 #wp20-events-filter label {
-	margin-right: 30px;
 	margin-bottom: 0;
 	padding: 0.7em 0; /* Must match `.wp20-events-query` (inherited from TwentySeventeen. */
+	text-align: right;
```

Comment:
does this have any effect?

Reply:
Yeah, but only when the text is very very long:

![Screenshot 2023-04-11 at 9 20 29 AM](https://user-images.githubusercontent.com/484068/231226339-a9ea9d7e-7086-4e21-b90f-1c9bd196e301.png)

![Screenshot 2023-04-11 at 9 20 44 AM](https://user-images.githubusercontent.com/484068/231226405-7b346a50-29d4-4073-a193-11a6b38024b7.png)

There's a good chance that'll never happen in practice, but I added it just to be safe.